day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,29 @@
+x = 10
+plural = "frogs"
+singular = "frog"
+
+while x > 0
+  if x > 2
+      puts "#{x} speckled #{plural} sat on a log
+        eating some most delicious bugs.
+        One jumped in the pool where it's nice and cool,
+        then there were #{x - 1} speckled #{plural}."
+    x -= 1
+    next
```

Comment:
Though using `next` is an ok stopgap measure for these conditionals, how might you re-order the sequence of if/elseif conditionals so that you don't need to use `next`?